examples/classic-kubernetes-multi-zone/variables.tf

```diff
@@ -77,7 +77,7 @@ variable "master_service_private_endpoint" {
 
 
 variable "worker_zones" {
```

Comment:
List of worker zones with zone as key & respective (public,private vlans) as values.